gdal/frmts/netcdf/netcdfsg.cpp

```diff
@@ -767,8 +794,17 @@ namespace nccfdriver
             // If matches, then establish a reference by placing this variable's data in both vectors
             if(!strcmp(contname, buf))
             {
-                char property_name[NC_MAX_NAME];
-                nc_inq_varname(this->nc, curr, property_name);
+                char property_name[NC_MAX_NAME] = {0};
```

Comment:
in most similar places we use NC_MAX_NAME+1

Reply:
Ooh yes, potential buffer overflow thanks :+1: